_templates/component/new/prompt.js

```diff
@@ -13,11 +13,17 @@ module.exports = [
   {
     type: 'input',
     name: 'parent',
-    message: "What is the component's parent? (Leave empty if not applicable)",
+    message:
+      "Does the component have an existing parent? If so, what's it's name? (Leave empty if not applicable)",
```

Comment:
Should be "what's its name" instead of "what's it's name"

Reply:
Yes